src/thing_creature.c

```diff
@@ -3830,12 +3830,12 @@ long player_list_creature_filter_dragging_specific_thing(const struct Thing *thi
         }
         return -1;
     }
-    if ((param->class_id > 0) || (param->model_id > 0) || (param->plyr_idx >= 0))
+    if ((param->class_id > 0) || ((param->model_id > 0) && !is_creature_model_wildcard(param->model_id)) || (param->plyr_idx >= 0))
```

Comment:
by function docstring it could be used to find an imp dragging trap box. So model check required.

Reply:
I don't get this, before it 'just' excluded -1 for creature_any, I caught that case now too right?

But I also don't know the specifics of what you mean, what model would need checking here?

Reply:
ok